test/unit/util/async-each-limit.test.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const { test } = require('tap')
+const sinon = require('sinon')
+const eachLimit = require('../../../lib/util/async-each-limit')
+
+test('eachLimit', async (t) => {
```

Comment:
this is prob a habit from mocha but you don't need this outer test since there's only one test